src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.WorkItems;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.WorkItems
+{
+    public class SarifWorkItemModel : WorkItemModel<SarifWorkItemContext>
+    {
+        public void InitializeFromLog(SarifLog sarifLog, SarifWorkItemContext context = null)
+        {
+            this.Context = context ?? new SarifWorkItemContext();
+
+            context.InitializeFromLog(sarifLog);
+
+            // Shared GitHub/Azure DevOps concepts
+
+            this.LabelsOrTags = new List<string> { $"default{nameof(this.LabelsOrTags)}" };
```

Comment:
>}; [](start = 89, length = 2)

Why is the default a list containing one string whose value is `"defaultLabelsOrTags"`? Why isn't the default an empty list, or null?

Reply:
Because this code is working, ie, labels are applied to both ADO and GH. These default values are helpful to verify this functionality functions.

---
In reply to: [381024282](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381024282) [](ancestors = 381024282)